db/migrate/20240205160647_remove_organization_sequential_id_index_from_invoices.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveOrganizationSequentialIdIndexFromInvoices < ActiveRecord::Migration[7.0]
+  def change
+    remove_index :invoices, name: 'unique_organization_sequential_id'
```

Comment:
Rails/ReversibleMigration: remove_index(without column) is not reversible. (https://rails.rubystyle.guide#reversible-migration, https://api.rubyonrails.org/classes/ActiveRecord/Migration/CommandRecorder.html)